docs/image-transfer/ExportPipelines/README.md

```diff
@@ -0,0 +1,33 @@
+# Export Pipeline
+
+Create an ExportPipeline resource in your source container registry. Note that the export will not take place until the export pipeline is run.
+
+## Create the managed identity
+
+Note: These examples are implemented with user-assigned identities. ExportPipelines and ImportPipelines also support SystemAssigned identities. If using system identity, please handle role assignment after the pipeline resources are created and prior to running.
+
+### Create the user-assigned managed identity
+
+```
+az identity create --resource-group myResourceGroup --name myPipelineId 
+```
+```
+principalID=$(az identity show --resource-group myResourceGroup --name myPipelineId --query principalId --output tsv)
+resourceID=$(az identity show --resource-group myResourceGroup --name myPipelineId --query id --output tsv)
+```
+
+### Grant the identity read access to KV
+
+```
+az keyvault set-policy --name mykeyvault \ --resource-group myResourceGroup \ --object-id $principalID \ --secret-permissions get
```

Comment:
Probably need to line break these.

Reply:
Is this where templates go? I thought Docs either had a location, or the ARM team had a location to post this content? 
@dlepow where do other docs reference templates?

Reply:
@sajayantony I removed the READMEs so we can just link to Dan's docs.